packages/modeling/src/operations/modifiers/reTesselateCoplanarPolygons.js

```diff
@@ -307,6 +308,7 @@ const reTesselateCoplanarPolygons = (sourcepolygons) => {
           const points2d = prevpolygon.outpolygon.rightpoints.concat(prevpolygon.outpolygon.leftpoints)
           const vertices3d = points2d.map((point2d) => orthobasis.to3D(point2d))
           const polygon = poly3.fromPointsAndPlane(vertices3d, plane) // TODO support shared
+          polygon.color = color
 
```

Comment:
This should be conditional, so you dont attach undefined on all polygons

Reply:
Thank you, and agreed. I've just added a commit.